benchmark/cublas/blas3/gemm.cpp

```diff
@@ -35,6 +35,11 @@ static inline void cublas_routine(args_t&&... args) {
   } else if constexpr (std::is_same_v<scalar_t, double>) {
     CUBLAS_CHECK(cublasDgemm(std::forward<args_t>(args)...));
   }
+#ifdef BLAS_ENABLE_HALF
+  else if constexpr (std::is_same_v<scalar_t, cl::sycl::half>) {
+    CUBLAS_CHECK(cublasHgemm(std::forward<args_t>(args)...));
+  }
+#endif
```

Comment:
```suggestion
  } else if constexpr (std::is_same_v<scalar_t, cl::sycl::half>) {
    CUBLAS_CHECK(cublasHgemm(std::forward<args_t>(args)...));
  }
```
I think this `#ifdef` is redundant. Same for the `rocblas` one.